test/integration/acceptance/create_asset_test.cpp

```diff
@@ -75,22 +80,24 @@ TEST_F(CreateAssetFixture, Basic) {
  * @then stateful validation failed
  */
 TEST_F(CreateAssetFixture, ExistingName) {
-  IntegrationTestFramework(1)
-      .setInitialState(kAdminKeypair)
-      .sendTxAwait(
-          makeUserWithPerms(),
-          [](auto &block) { ASSERT_EQ(block->transactions().size(), 1); })
-      .sendTx(complete(baseTx().createAsset(kAssetName, kDomain, kPrecision)))
-      .checkProposal(
-          [](auto &proposal) { ASSERT_EQ(proposal->transactions().size(), 1); })
-      .checkVerifiedProposal(
-          // todo igor-egorov, 2018-08-15, IR-1625, add precise check of failure
-          // reason
-          [](auto &vproposal) {
-            ASSERT_EQ(vproposal->transactions().size(), 0);
-          })
-      .checkBlock(
-          [](auto block) { ASSERT_EQ(block->transactions().size(), 0); });
+  for (auto const type : storage_types)
```

Comment:
It would be better to let GTest know we run test with different parameter through instantiating GTEST SUITE.